en/dronecan/cuav_can_pmu.md

```diff
@@ -1,13 +1,18 @@
 # CAUV CAN PMU
 
-CAN PMU<sup>&reg;</sup> is a high-precision [UAVCAN](../uavcan/README.md) power module developed by CUAV<sup>&reg;</sup>.
+CAN PMU<sup>&reg;</sup> is a high-precision [DroneCAN](README.md) power module developed by CUAV<sup>&reg;</sup>.
 It runs the CUAV ITT compensation algorithm, which enables drones to get the battery data more accurately.
 
 ![CAN PMU](../../assets/hardware/power_module/cuav_can/can_pmu.jpg)
 
 It is recommended for use in large commercial vehicles, but might also be used for research vehicles.
 
-## Specifications
+## Where to Buy
```

Comment:
Hmmm. I don't have a problem with Where to Buy, but we are now inconsistent. I might use this PR to fix them all up.